src/shared/services/form/form-validators.service.spec.ts

```diff
@@ -24,4 +24,15 @@ describe('FormValidatorsService', () => {
     result.updateValueAndValidity();
     expect(result.invalid).toBeTruthy();
   });
+
+  it('should validate text field for MANDATORY with regular expression', () => {
```

Comment:
Shouldn't we have tests that would verify what is asked in the ticket? This would be a field with value containing spaces?

Also a review of all tests according to the rules that are expected to be preserved would also be useful.

Reply:
Done.

Reply:
Actually I am not sure that it works properly. Just ran a test like this one:

```
  fit('should validate text field for MANDATORY with regular expression', () => {
    let formControl: FormControl = new FormControl();
    let caseField: CaseField = aCaseField('id', 'label', 'Text', 'MANDATORY', null);
    caseField.field_type.regular_expression = '^([Pp][Bb][Aa])\d{7}$'
    let result: FormControl = formValidatorsService.addValidators(caseField, formControl);
    
   
    result.setValue('PBA 1234567');
    result.markAsTouched();
    result.updateValueAndValidity();
    expect(result.invalid).toBeTruthy();
  });
```

which should pass but is failing. The space should make the test fail as far as I understood the ACs?

Reply:
As discussed.